src/manual-multiple-soulmates/test.json

```diff
@@ -1,5 +1,7 @@
 {
   "ViewAll": "View all",
+  "NumberofCards": "4",
```

Comment:
Let's ask if we can actually get rid of this attribute. It doesn't seem they used it ever.